test/extensions/filters/http/ext_proc/ext_proc_integration_test.cc

```diff
@@ -626,6 +629,41 @@ TEST_P(ExtProcIntegrationTest, GetAndCloseStream) {
   verifyDownstreamResponse(*response, 200);
 }
 
```

Comment:
I think `RequestMessageTimeout` test case in this file will hit the path I commented above

Do you want to add `RequestMessageTimeoutWithTracing` test case?